src/patterns/components/dictionary/collection.yaml

```diff
@@ -0,0 +1,9 @@
+notes: |
+  This spark-extras pattern is intended for displays of key-value pairs.
```

Comment:
We need something to visually separate things like package names so they stand out form the rest of the paragraph. There is the class `drizzle-CodeHighlight` that puts a grey background behind it, maybe that?

Reply:
Also, we should avoid using "pattern". Most of the time "component" should be used instead.

Reply:
@yodasw16  Rob and I talked yesterday about having a badge or label for the package names so users can see quickly at a glance what package things are for.